### PR TITLE
Add Python3 support.

### LINKED ISSUE #15: Add python3 support
cf https://github.com/novafloss/django-x509/blob/master/.travis.yml#L3

### PR DESCRIPTION
**Pull request by Natim**
Fixes #15

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,13 +17,16 @@ def read_relative_file(filename):
...
-                     'Programming Language :: Python',
-                     'Programming Language :: Python :: 2.7',
-                     'Programming Language :: Python :: 3'],
+        classifiers=[
+            'Development Status :: 5 - Production/Stable',
+            'Environment :: Web Environment',
+            'Intended Audience :: Developers',
+            'License :: OSI Approved :: MIT License',
+            'Programming Language :: Python',
+            'Programming Language :: Python :: 2.7',
```
Comment (wo0dyn) [inline setup.py]: Do we need to add `'Programming Language :: Python :: 2',` prior to 2.7? Not sure.

File: x509/django/models.py [hunk 2]
```diff
@@ -5,9 +5,10 @@
 from x509.django.compat import UUIDField
 
 try:
-    from django.contrib.contenttypes.generic import GenericForeignKey
-except ImportError:
     from django.contrib.contenttypes.fields import GenericForeignKey
+except ImportError:
+    # Django 1.8
+    from django.contrib.contenttypes.generic import GenericForeignKey
```
Comment (wo0dyn) [inline x509/django/models.py]: :+1:

File: setup.py (hunk 1 above)
Comment (Natim) [inline setup.py]: We can sure.

Comment (wo0dyn) [inline setup.py]: Thanks. :+1:

**Pull request merged**